doc/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender/index.md

```diff
@@ -0,0 +1,442 @@
+---
+layout: page
+title: Flume log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender
+sort_idx: 50
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Flume log appender encapsulates received logs into Flume events and sends these events to external Flume sources via Avro RPC.
+
+# Creating Flume log appender in Admin UI
+
+The easiest way to create a Flume log appender for your application is by using Admin UI.
+
+To create a log appender do the following:
+
+1.In the **Log appenders** window, click **Add log appender**. <br/>
```

Comment:
This looks like a cheat to make the text look like a list with the numeration provided. But, as this does not match to a list, it does not follow the list CSS styling and will look badly should the style for the list be changed.

Could this be made a list rather than text?